iiify/search.py

```diff
@@ -51,8 +46,7 @@ def iiif_search(identifier, query):
 
         for box in paragraph['boxes']:
             x = int(box['l'])
-            y = int (box['t'])
-            right = 0
+            y = int(box['t'])
             # If r is missing then use the paragraph
```

Comment:
Should right =0 be removed here? Its used later and only set in the if statement.